### PR TITLE
414: Venue template style

### PR DESCRIPTION
**Pull request by maxb94**

### CODE REVIEW

File: src/settings.ts [hunk 1]
```diff
@@ -5,7 +5,8 @@ export type TemplateType =
   | "ZOOM_ROOM"
   | "THEME_CAMP"
   | "PERFORMANCE_VENUE"
-  | "ART_PIECE";
+  | "ART_PIECE"
+  | "ARTCAR";
```
Comment (antoineeudes) [inline src/settings.ts:9]: ART_CAR to be coherent with the rest

File: src/pages/Account/Venue/TemplateForm.tsx [hunk 2]
```diff
@@ -1,29 +1,81 @@
...
+              next={next}
+            />
+          </div>
+        </div>
+      </div>
+      <div className="page-side">
+        {selectedTemplate && (
+          <img
+            src={templateImageMap[selectedTemplate.type]}
+            alt={"venue"}
```
Comment (antoineeudes) [inline src/pages/Account/Venue/TemplateForm.tsx:43]: "venue"

File: src/pages/Account/Venue/Venue.scss [hunk 3]
```diff
@@ -5,22 +5,119 @@ $vertical-padding: 20px;
...
+}
+
+.page-side {
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  .page-container-left {
+    background-color: $black;
+    padding-top: 40px;
+    overflow-y: scroll;
```
Comment (antoineeudes) [inline src/pages/Account/Venue/Venue.scss:23]: I prefer auto

File: src/scss/constants.scss [hunk 4]
```diff
@@ -13,3 +13,9 @@ $yellow: #ffe600;
 $light-green: #3ce348;
 $dark-green: #208a28;
 $stripe-auto-fill-background-color: #fefde5;
+
+$primary: #7c46fb;
+$black: #000000;
```
Comment (mathieud) [inline src/scss/constants.scss:18]: We had black already

File: src/scss/constants.scss [hunk 5]
```diff
@@ -13,3 +13,9 @@ $yellow: #ffe600;
 $light-green: #3ce348;
 $dark-green: #208a28;
 $stripe-auto-fill-background-color: #fefde5;
+
+$primary: #7c46fb;
```
Comment (mathieud) [inline src/scss/constants.scss:17]: Isn't this in $primaryColor by default?

File: src/scss/constants.scss [hunk 6]
```diff
@@ -13,3 +13,9 @@ $yellow: #ffe600;
...
+$black: #000000;
+$white: #fff;
```
Comment (mathieud) [inline src/scss/constants.scss:19]: Exists already

File: src/scss/constants.scss (hunk 5 above)
Comment (mathieud) [reply src/scss/constants.scss:17]: Nevermind, it's not.

**Pull request merged**